docs/agents/sematext-agent/containers/metrics.md

```diff
@@ -15,13 +15,10 @@ The following table recaps all the metrics shipped by the agent.
 | cache memory          | Is the amount of memory allocated for page cache. The total amount of memory consumed by a container is the sum of RSS + cache memory |   
 | memory fail counter   | Is the number of times that memory cgroup limit was exceeded |
 | memory limit          | Designates the max allowed memory limit for the container cgroup  |
-| memory soft limit     | Represents the initial memory reservation for the container |
 | memory pages in       | The number of events each time the page is accounted to the container cgroup|
 | memory pages out      | Is the number of events each time a page is unaccounted from the container cgroup |
-| memory pages fault minor   | Represents the number of minor page faults accounted the cgroup |
-| memory pages fault major   | Represents the number of major page faults accounted the cgroup |
```

Comment:
Hm, what happened to all these metrics? We lost major/minor + some more? In other files too.

Reply:
Not sure what happened here. I'll submit a separate PR.